src/gui/guiFormSpecMenu.cpp

```diff
@@ -781,6 +782,54 @@ void GUIFormSpecMenu::parseImage(parserData* data, const std::string &element)
 	errorstream<< "Invalid image element(" << parts.size() << "): '" << element << "'"  << std::endl;
 }
 
+void GUIFormSpecMenu::parseAnimatedImage(parserData *data, const std::string &element)
+{
+	std::vector<std::string> parts = split(element, ';');
+
+	if ((parts.size() == 3) ||
+		((parts.size() > 3) && (m_formspec_version > FORMSPEC_API_VERSION)))
+	{
+		std::vector<std::string> v_pos   = split(parts[0], ',');
+		std::vector<std::string> v_geom  = split(parts[1], ',');
+		std::string name = unescape_string(parts[2]);
+
+		MY_CHECKPOS("animated_image", 0);
+		MY_CHECKGEOM("animated_image", 1);
+
+		v2s32 pos;
+		v2s32 geom;
+
+		if (data->real_coordinates) {
+			pos = getRealCoordinateBasePos(v_pos);
+			geom = getRealCoordinateGeometry(v_geom);
+		} else {
+			pos = getElementBasePos(&v_pos);
+			geom.X = stof(v_geom[0]) * (float)imgsize.X;
+			geom.Y = stof(v_geom[1]) * (float)imgsize.Y;
+		}
```

Comment:
I'd not support the old image pos and geom format or what that is.

Reply:
So just to confirm, you're suggesting that we just always use real coordinates since this is a new control?

Reply:
Yes, I think so.

Reply:
The result looks weird, but the old non-real coordinates result also looks weird and real coordinates are preferable anyway so I've pushed the change.